### DISCUSSION TITLE
Issues Re-verifying :)

### DISCUSSION THREAD

**Discussion by Chew** [community/community#50345] [GitHub Education]
Hi, the time has come for me to re-verify my academic status. I've been able to re-verify 3 times before, but this one is giving me extra trouble.

Our ID doesn't have a "Valid until" that GitHub is looking for. The document I submitted, which was a transcript, wasn't fully accepted. I was told to do three things:

> You are significantly more likely to be verified if you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name as it appears in your academic affiliation document plus a short bio.

The transcript still uses my deadname (/ legal name), which I refuse to set as my profile name. If it needs a last name, I can enter that, but the first name just isn't possible.

> Have you completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with all your relevant information, such as your full name as it appears in your image and a short bio?

See above, but I'm also in our school's student org, @utastudents, so I hope that helps or matters at all?

> Try applying again after making more [contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/viewing-contributions-on-your-profile#what-counts-as-a-contribution) to the GitHub Classroom assignment(s) created by your teacher.

I've made several contributions to a GitHub Classroom repository (specifically all mine in the @dillhoffuta org, I even have a public repo for it https://github.com/dillhoffuta/git-github-fundamentals-Chew)

Are there any ways to go through with this? Or another way to verify? I still very much have a lot more school work to do :p

**Comment by aviiciii** [comment]
Same issue for me :(
